pkg/testutil/util_new.go

```diff
@@ -92,7 +92,7 @@ func NewFS() *fileservice.FileServices {
 		panic(err)
 	}
 	fs, err := fileservice.NewFileServices(
-		"local",
+		"",
```

Comment:
why we replace "local" with ""?   Seems "local" is perfectly ok name, better than "".

Reply:
"" means no default, and will panic if use the FileServices instance without Get.